frontend/src/AlertsManagement/History.js

```diff
@@ -25,54 +47,68 @@ class History extends Component {
-        >
-          <ListItemText id={alertId}
-            disableTypography
-            primary={<Typography variant="body1" >Alert on 
-              <Box fontWeight='fontWeightBold' display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                {` ${alert.timestampDate} (${formatDate(alert.timestampDate)})`}
-              </Box> has
-              <Box display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                {` ${alert.anomalies.length} anomalies`}
-              </Box> with status
-              <Box display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                {` ${alert.status}`}
-              </Box>
-              </Typography>
-            }
-          />
-        </ListItem>
-      );
-    });
-    return alertItems;
-  }
-
   render() {
     const styles = ({
       root: {
         flexGrow: 1,
-        maxWidth: 900,
+        maxWidth: '90%',
         margin: 'auto',
       },
+      header: {
+        marginTop: '50px',
+        marginBottom: '30px'
+      },
+      chip: {
+        margin: '5px',
+      }
     });
+
     const { allAlerts } = this.state;
     
-    if (allAlerts === null) {
-      return <div />;
-    }
+    if (allAlerts === null) return <div />;
 
     return (
       <div style={styles.root}>
-        <Typography variant="h4" gutterBottom style={{ marginTop: '50px'}}>Alerts Archive</Typography>
-        <List className="all-alerts-list">
-          {this.createListItems(allAlerts)}
-        </List>
+        <Typography variant="h4" gutterBottom style={styles.header}>Alerts Archive</Typography>
+        
+        <MaterialTable icons={tableIcons}
+          actions={[
+            {
+              icon: () => <LinkIcon />,
+              tooltip: 'See More',
+              onClick: (_event, rowData) => this.props.history.push(`alerts/${rowData.id}`),
+            }
+          ]}
+          columns={[
+            { title: 'Timestamp', field: 'timestampDate', filtering: false },
+            { title: '# of Anomalies', field: 'numAnomalies' },
+            { title: 'Status', field: 'status', sorting: false },
+            { title: 'Priority', field: 'priority', type: 'numeric', align: 'left' },
+            { 
+              title: 'Metrics', field: 'metrics', sorting: false,
+              render: rowData => rowData.metrics.split(DATA_DELIMITER).map((metric) => 
+                <Chip label={metric} color="secondary" variant="outlined" 
+                  size="small" style={styles.chip}
+                />)
+            },
+            { 
+              title: 'Dimensions', field: 'dimensions', sorting: false,
+              render: rowData => rowData.dimensions.split(DATA_DELIMITER).map((dimension) => 
+                <Chip label={dimension} color="primary" variant="outlined" 
+                  size="small" style={styles.chip}
+                />)
+            }
+          ]}
+          data={rows(this.state.allAlerts)} 
+          options={{
+            exportButton: true,
```

Comment:
ooc, are the `exportButton` and `search` bar built in components of `MaterialTable`?
did you need to add any additional code for those features?

Reply:
yeah they are built in! :D

Reply:
wow that is pretty convenient